### PR TITLE
Tag importer

### LINKED ISSUE #81: Add a step to the image import process that loads a CSV of existing tag information from Tandem Vault
Tandem Vault provides a download link in its admin for all existing Tandem Vault tags and their synonyms.  Unfortunately it doesn't look like this download can be retrieved programmatically, so we'll need to add a step to the import process that allows a manually-downloaded CSV to be passed in and processed.

### PR DESCRIPTION
**Pull request by cjg89**
Added an importer for existing tags and their synonyms from Tandem Vault.  Tag data must be downloaded manually from the Tandem Vault admin and passed to the import command.

Resolves #81

### CODE REVIEW

File: images/management/commands/import-tandemvault-tags.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        return
+
+    '''
+    Load a CSV of Tandem Vault tags + synonyms.
+    '''
+    def load_tandemvault_tags(self):
+        try:
+            csv_reader = csv.DictReader(self.tandemvault_tags_csv)
+        except csv.Error:
+            logging.error(
```
Comment (jmbarne3) [inline images/management/commands/import-tandemvault-tags.py]: Is there an inner exception or message we can print to give us a clue what the problem with the CSV is?

File: images/management/commands/import-tandemvault-tags.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+                self.tags_updated += 1
+
+            # Remove existing Tandem Vault synonym relationships
+            existing_synonyms = tag.synonyms.filter(source=self.source)
+            if existing_synonyms:
+                tag.synonyms.remove(*existing_synonyms)
+
+            # Assign fresh Tandem Vault synonym relationships
+            for synonym in synonyms:
+                s, created = ImageTag.objects.get_or_create(name=synonym, source=self.source)
```
Comment (jmbarne3) [inline images/management/commands/import-tandemvault-tags.py]: Should we track and print out in the stats the number of synonyms added to give some insight to this portion of the importer?

File: images/management/commands/import-tandemvault-tags.py (hunk 1 above)
Comment (cjg89) [reply images/management/commands/import-tandemvault-tags.py]: Updated CSV error handling to spit out errors coming from the CSV lib, and added some missing sanity checks to make sure the file looks like a CSV before looping through it.

File: images/management/commands/import-tandemvault-tags.py (hunk 2 above)
Comment (cjg89) [reply images/management/commands/import-tandemvault-tags.py]: Added.

**Pull request merged**